src/Symfony/Component/HttpFoundation/Tests/File/MimeType/MimeTypeTest.php

```diff
@@ -14,17 +14,16 @@
 use Symfony\Component\HttpFoundation\File\MimeType\MimeTypeGuesser;
 use Symfony\Component\HttpFoundation\File\MimeType\FileBinaryMimeTypeGuesser;
 
+/**
+ * @requires extension fileinfo
+ */
 class MimeTypeTest extends \PHPUnit_Framework_TestCase
 {
     protected $path;
 
     public function testGuessImageWithoutExtension()
     {
-        if (extension_loaded('fileinfo')) {
-            $this->assertEquals('image/gif', MimeTypeGuesser::getInstance()->guess(__DIR__.'/../Fixtures/test'));
-        } else {
-            $this->assertNull(MimeTypeGuesser::getInstance()->guess(__DIR__.'/../Fixtures/test'));
```

Comment:
When the extension isn't loaded, the guesser does not return null but throws a LogicException here. This was dead code.